ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.16-nightly-x86.yaml

```diff
@@ -24,8 +24,9 @@ tests:
       BASE_DOMAIN: perfscale.hyp.azure.rhperfscale.org
       BASE_DOMAIN_RESOURCE_GROUP: perfscale-prow
       COMPUTE_NODE_TYPE: Standard_D4s_v5
-      CONTROL_PLANE_INSTANCE_TYPE: Standard_D16s_v3
-      CUSTOM_AZURE_REGION: centralus
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D16s_v5
+      CUSTOM_AZURE_REGION: eastus
```

Comment:
Based on this readout for ARO [URL](https://docs.google.com/presentation/d/1OMtiM6S0yYCcE-sUeSNYfbSyZe8EtjQ2FGu-hqLdul4/edit#slide=id.gc52980a47d_1_0), please update:

- Control Plane Node: Standard_D32s_v5 [for large scale cluster]
- Infra Nodes: Standard_E16s_v3 [with v5 instances we observed packet drops between infra <--> compute nodes]

Reply:
I did the update.

Reply:
Attention here about the AWS settings:
For 120 nodes [SIZE_VARIANT: large](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml#L48) is set
SIZE_VARIANT: [large](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L21) means * "large" - 16 vCPU, 64GB+ control plane nodes, default workers, suitable for clusters up to 250 nodes

Reply:
I had a PR https://github.com/openshift/release/pull/44600 to synchronize instance type/sizes across Self-Managed vs Managed in AWS, in-order to get relevant signaling from same instances. Helps in establishing confidence if performance issues arise only in one or both.

That said, it would be good to have ARO & Self-Managed Azure generating data on equivalent instance types rather than different one.

Reply:
@krishvoor In your new pr https://github.com/openshift/release/pull/57655#pullrequestreview-2362212531, for 120 nodes, you used SIZE_VARIANT: large. SIZE_VARIANT: [large](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L21) means * "large" - 16 vCPU, 64GB+ control plane nodes
So do you agree to use Standard_D16s_v5 for Azure so that it is same to AWS?

Reply:
Use Standard_D16s_v5 to be same size as https://github.com/openshift/release/pull/57655 control-plane-ipsec-120nodes which uses r5.4xlarge on AWS